linera-rpc/proto/rpc.proto

```diff
@@ -221,6 +224,28 @@ message Certificate {
   bytes blobs = 7;
 }
 
+message CertificateValue {
```

Comment:
In `Certificate` below, the value is just `bytes`. We could do that here, too, and not distinguish between the variants.